Makefile

```diff
@@ -1,24 +1,27 @@
-SHORT_NAME := slugbuilder
+SHORT_NAME ?= slugbuilder
 
 export GO15VENDOREXPERIMENT=1
 
 # Note that Minio currently uses CGO.
 
-VERSION := 0.0.1-$(shell date "+%Y%m%d%H%M%S")
+VERSION ?= 0.0.1-$(shell date "+%Y%m%d%H%M%S")
 LDFLAGS := "-s -X main.version=${VERSION}"
+IMAGE_PREFIX ?= deis
 BINDIR := ./rootfs/bin
 DEV_REGISTRY ?= $$DEV_REGISTRY
-DEIS_REGISTRY ?= ${DEV_REGISTRY}
+DEIS_REGISTRY ?= ${DEV_REGISTRY}/
 
-IMAGE := ${DEIS_REGISTRY}/${SHORT_NAME}:${VERSION}
+IMAGE := ${DEIS_REGISTRY}${IMAGE_PREFIX}/${SHORT_NAME}:${VERSION}
```

Comment:
Mea culpa: I started adding the slash elsewhere so that doing `DEV_REGISTRY=''` would remove the repository prefix altogether and thus work with Docker Hub (which lives at docker.io but for some reason won't accept that as a repository prefix on `docker push`.)

Hacky bash solution, if someone knows of a cleaner way to build up `$IMAGE` with or without a repo prefix, let's do that.